spyderlib/spyder.py

```diff
@@ -1718,17 +1669,17 @@ def setup_default_layouts(self, index, settings):
         self._custom_layout_timer.start(100)

```

Comment:
It works for me if I comment these lines related to `_custom_layout_timer`. With this change I can prevent the window overflowing issue in my machine :-)